test/helper.lua

```diff
@@ -323,13 +323,59 @@ function helpers.tarantool_version_at_least(wanted_major, wanted_minor,
     return true
 end
 
-function helpers.update_cache(cluster, space_name)
+function helpers.update_sharding_key_cache(cluster, space_name)
     return cluster.main_server.net_box:eval([[
-        local sharding_key = require('crud.common.sharding_key')
+        local sharding_metadata = require('crud.common.sharding.sharding_metadata')
 
         local space_name = ...
-        return sharding_key.update_cache(space_name)
+        return sharding_metadata.update_sharding_key_cache(space_name)
     ]], {space_name})
 end
```

Comment:
`require('crud.common.sharding_key').update_cache()` should work (verbatim). At least one user rely on it currently.

Reply:
Updated.